linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/set_iptables_outbound_n_established_connections_configured/tests/no_output_rules.fail.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I see test scenarios but I don't any check in this rule, making the test scenarios useless. Would you like to include a SCE check for this rule? Maybe it would be more interesting than removing the tests.